include/jemalloc/internal/arena_inlines_b.h

```diff
@@ -310,7 +319,7 @@ arena_dalloc(tsdn_t *tsdn, void *ptr, tcache_t *tcache,
 	}
 }
 
-static inline void
+JEMALLOC_ALWAYS_INLINE void
```

Comment:
Why are we making these slow paths always inline?

Reply:
ah that wasn't intentional. will fix.